docs/security/using-fail2ban-for-security.md

```diff
@@ -126,11 +126,14 @@ Fail2ban reads its configuration files so that all `.conf` files are read first
 
         cd /etc/fail2ban
 
-2.  Copy `fail2ban.conf` to `fail2ban.local`:
+2.  The file `fail2ban.conf` contains the default configuration variables for Fail2ban logging, the socket used to communicate with the daemon, and the location of the PID file. Changes should be made in a separate file, `fail2ban.local`, which overrides `fail2ban.conf`. If you want, you can copy `fail2ban.conf` to `fail2ban.conf`, commenting out all variables, and then uncomment only the options you want to modify:
 
         cp fail2ban.conf fail2ban.local
+        sed -i 's/^/#/g' fail2ban.local
```

Comment:
kinda would work if people wouldn't also forget to uncomment section names ;-)  and should provide remedy for the issue of stale configurations propagating across releases
would I use such approach myself?  probably not since it is hard to see what I am actually changing without running some 'grep' command, people would tend to uncomment more than necessary to achieve the goal, etc